chore: unit test regression

Related to https://github.com/kytos-ng/of_core/issues/128

```
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_k_shortest_paths_among_users[User1-User2] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_k_shortest_paths_among_users[User1-User3] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_k_shortest_paths_among_users[User1-User4] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_k_shortest_paths_among_users[User2-User3] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_k_shortest_paths_among_users[User2-User4] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_k_shortest_paths_among_users[User3-User4] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_constrained_k_shortest_paths_among_users[User1-User2] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_constrained_k_shortest_paths_among_users[User1-User3] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_constrained_k_shortest_paths_among_users[User1-User4] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_constrained_k_shortest_paths_among_users[User2-User3] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_constrained_k_shortest_paths_among_users[User2-User4] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_constrained_k_shortest_paths_among_users[User3-User4] - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_cspf_delay_spf_attribute_between_u1_u4 - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_cspf_reliability_between_u1_u2 - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_cspf_bandwidth_between_u1_u4 - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_cspf_delay_between_u2_u3 - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_cspf_ownership_between_s4_s6 - assert []
FAILED tests/integration/test_paths_edges.py::TestPathsEdges::test_cspf_flexible_between_s4_s6 - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_user_k1 - assert 0 == 1
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_user_k2 - assert 0 == 2
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_user_k4 - assert 0 == 4
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_switch - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_switch_switch - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_user_t1 - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_user_t2 - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_user_switch_t2 - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_switch_switch_t2 - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_reliability - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_cspf_with_multiple_owners - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_reliability_detailed - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_delay - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_bandwidth_detailed - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_bandwidth_detailed_t2 - assert []
FAILED tests/integration/test_paths_metadata.py::TestPathsMetadata::test_path_constrained_bandwidth_delay - assert []
FAILED tests/integration/test_paths_simple.py::TestPathsSimple::test_path_s1_s2 - assert []
FAILED tests/integration/test_paths_simple.py::TestPathsSimple::test_path_to_self - assert []
FAILED tests/integration/test_paths_simple.py::TestPathsSimple::test_path_s1_s3_constrained_blue - assert []
FAILED tests/integration/test_paths_simple.py::TestPathsSimple::test_path_s1_s2_constrained_not_owner_red - assert []
FAILED tests/integration/test_paths_simple.py::TestPathsSimple::test_path_s1_s2_constrained_not_owner_blue - assert []
FAILED tests/integration/test_paths_simple.py::TestPathsSimple::test_path_s1_s3_constrained_bandwidth - assert []
```

--- a/graph/graph.py
+++ b/graph/graph.py
@@ -15,7 +15,7 @@ try:
     import networkx as nx
     from networkx.exception import NetworkXNoPath, NodeNotFound
 except ImportError:
-    PACKAGE = "networkx==2.5.1"
+    PACKAGE = "networkx==2.8.6"
     log.error(f"Package {PACKAGE} not found. Please 'pip install {PACKAGE}'")
 
 

--- a/graph/graph.py
+++ b/graph/graph.py
@@ -11,12 +11,8 @@ from .filters import EdgeFilter, ProcessEdgeAttribute, TypeCheckPreprocessor, Ty
 from .weights import (nx_edge_data_delay, nx_edge_data_priority, nx_edge_data_weight)
 
 
-try:
-    import networkx as nx
-    from networkx.exception import NetworkXNoPath, NodeNotFound
-except ImportError:
-    PACKAGE = "networkx==2.8.6"
-    log.error(f"Package {PACKAGE} not found. Please 'pip install {PACKAGE}'")
+import networkx as nx
+from networkx.exception import NetworkXNoPath, NodeNotFound
 
 
 class KytosGraph:

--- a/tests/integration/test_paths.py
+++ b/tests/integration/test_paths.py
@@ -17,7 +17,6 @@ class TestPaths:
             "generate_topology" if not val else "generate_topology_" + str(val)
         )
         method = getattr(self, method_name)
-        method()
         switches, links = method()
 
         self.graph = KytosGraph()
@@ -37,6 +36,7 @@ class TestPaths:
         """Add a new switch to the list of switches"""
         switch = Switch(name)
         switch.is_active = lambda: True
+        switch.is_enabled = lambda: True
         switches[name] = switch
 
     @staticmethod

